common/src/main/java/com/sequenceiq/cloudbreak/telemetry/monitoring/MonitoringUrlResolver.java

```diff
@@ -14,12 +18,13 @@ public class MonitoringUrlResolver {
 
     private final String monitoringPaasEndpointConfig;
 
-    public MonitoringUrlResolver(String monitoringEndpointConfig, String monitoringPaasEndpointConfig) {
-        this.monitoringEndpointConfig = monitoringEndpointConfig;
-        if (StringUtils.isBlank(monitoringPaasEndpointConfig)) {
-            this.monitoringPaasEndpointConfig = monitoringEndpointConfig;
+    @Inject
+    public MonitoringUrlResolver(MonitoringConfiguration monitoringConfiguration) {
```

Comment:
Might worth to reconsider the 2 different RemoteWrite and PaasRemoteWriteUrl distinction as it might become obsolete quite soon.

Reply:
I would do it in a separate ticket to clean up saas related configs.